lib/dry/configurable/setting.rb

```diff
@@ -114,6 +112,11 @@ def clonable_value?
         CLONABLE_VALUE_TYPES.any? { |type| value.is_a?(type) }
       end
 
+      # @api private
+      def evaluate
```

Comment:
I did change the scope to public, in order to have the evaluation available at config.rb